README.md

```diff
@@ -56,6 +56,8 @@ As a result, you should see LEDs blinking on the board.
 >export ZEPHYR_BASE=<path_to_zephyr_directory>
 >```
 
+If you want to run a custom application, refer to the [nPM1300 documentation][nPM1300_documentation] to learn how to properly configure the required setup.
```

Comment:
Perhaps a link to npmx docs should be also added here? How about:
`In order to create a custom application, refer to npmx docs for API usage and nPM1300 documentation for HW setup and PMIC HW functionalities.`

Reply:
```suggestion
If you want to create a custom application, refer to the following guidelines:
- [link to npmx documentation] for API usage
- [nPM1300 documentation][nPM1300_documentation] for the required setup configuration
```